.circleci/scripts/automatePostSteps.py

```diff
@@ -190,4 +246,69 @@ def validateLogs():
 			print ("Migration test failed.. please go through the eet logs")
 			print ("{}/migrationPostStepsTest.log".format(test_clients_path))
 
-validateLogs()
+
+#----------------------------------------------------------------------------------------------------------------------------------------------------------#
+#-------------------------------------------------------------- Main Migration Workflow -------------------------------------------------------------------#
+
+# The main steps to validate the migration
+if __name__ == '__main__':
+	msg = ""
+	if TEST_RESTART:
+		msg = "Will do migration test for {} with restart steps".format(NETWORKUSED)
+	else:
+		msg = "Will do migration test for {}".format(NETWORKUSED)
+	print(msg)
+
+	cleanLogs()
+	NO_OF_NODES=validateInputs()
+	print("No of nodes required : {}".format(NO_OF_NODES))
+
+	with open("{}/terraform/deployments/ansible/inventory/{}".format(INFRASTRUCTURE_REPO , INVENTORY), 'r') as fin:
+		print(fin.read())
+
+	buildSavedZip()
+	shutil.copy("saved.zip", "{}/terraform/deployments/ansible/roles/hgc-deploy-psql-mig/files/".format(INFRASTRUCTURE_REPO))
+
+	migrationCmd = "ansible-playbook -i ./inventory/{} --private-key {} -u ubuntu -e branch={} -e app_dir={} -e enable_newrelic=false -e hgcapp_service_file=hgcappm410NR -e saved_round_number={} play-deploy-migration.yml".format(INVENTORY, PEM_FILE, BRANCH_NAME, SERVICES_REPO, SIGNEDSTATE)
```

Comment:
Leo, lets change the location of this log `migrationPostStepsTest.log`. Remember it was hard to navigate to the location of this log in `repo/infrastructure/terraform/deployments/ansible` . Lets move it repo by giving it a absolute path. like `/repo/migrationPostStepsTest.log`. Not a huge deal though.

Reply:
Make sense. I will move it to the /repo level.